assignments/objects.js

```diff
@@ -18,27 +18,66 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy@psu.edu",
+  gender: "F"
+}
+
+const kennan = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function() {
+    return "Hello, my name is Kennan!";
+  }
+}
+
+const keven = {
+  id: 3,
+  name: "Keven",
+  email: "kmummery2@wikimedia.org",
+  gender: "M"
+}
 
+const gannie = {
+  id: 4,
+  name: "Gannie",
+  email: "gmartinson3@illinois.edu",
+  gender: "M"
+}
+
+const antonietta = {
+  id: 5,
+  name: "Antoinetta",
+  email: "adaine5@samsung.com",
+  gender: "F",
+  multiplyNums: function(num1, num2) {
+    return num1 * num2;
+  }
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(mitzi.name);
 // Kennan's ID
-
+console.log(kennan.email);
 // Keven's email
-
+console.log(keven.id);
 // Gannie's name
-
+console.log(gannie.name);
 // Antonietta's Gender
-
+console.log(antonietta.gender);
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log(kennan.speak());
```

Comment:
This works just fine, but you can alternatively define the `kennan.speak` method here instead of directly in the kennan object, like so: 
```suggestion
kennan.speak = function() {
  return "Hello, my name is Kennan!";
}
console.log(kennan.speak());
```

Reply:
✔I realized that later! thanks